samcli/commands/list/cli_common/options.py

```diff
@@ -0,0 +1,34 @@
+"""
+Common CLI options shared by various commands
+"""
+
+import click
+
+
+def stack_name_click_option():
+    return click.option(
+        "--stack-name",
+        help=(
+            "Name of corresponding deployed stack.(Not including "
+            "a stack name will only show local resources defined "
+            "in the template.) "
+        ),
+        type=click.STRING,
+    )
+
+
+def stack_name_option(f):
+    return stack_name_click_option()(f)
+
+
+def output_click_option():
+    return click.option(
+        "--output",
```

Comment:
I feel that --output as a common option is too general to have default as table, I would move this into the command if this option will not be shared with other commands. If the output option will be shared with other commands, I don't feel that table would be an ideal default option, as other commands most likely have default output as console/json

Reply:
This is only common to list options, all of which today make sense to default to table.

Reply:
Oh I see, for list options you created a new common file to store all the list only common options. My bad I misunderstood the structure